core/extension/index.js

```diff
@@ -1,14 +1,12 @@
 /*---------------------------------------------------------------------------------------------
- * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
- * See LICENSE.md in the project root for license terms and full copyright notice.
- *--------------------------------------------------------------------------------------------*/
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
 
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis
 const globalSymbol = Symbol.for("itwin.core.frontend.globals");
-const ext = globalThis[globalSymbol].getExtensionApi("import.meta.url");
+const ext = globalThis[globalSymbol].getExtensionApi(import.meta.url);
```

Comment:
does this work now with the change to the typings being "module"?

Also with that change to "module" is it required to use esmodules?

Reply:
yes `import.meta.url` would only work if we deliver esmodules (setting the type to "module")
considering at the moment this pkg is supposed to help augment frontend applications, i think its safe to start pushing only delivering esm for this pkg.